client/src/containers/App.tsx

```diff
@@ -85,83 +85,126 @@ const WhoOwnsWhatRoutes: React.FC<{}> = () => {
   );
 };
 
-export default class App extends Component<Props, State> {
-  constructor(props: Props) {
-    super(props);
+const getMainNavLinks = () => {
+  const paths = createWhoOwnsWhatRoutePaths();
+  return [
+    <LocaleNavLink exact to={paths.home} key={1}>
+      <Trans>Home</Trans>
+    </LocaleNavLink>,
+    <LocaleNavLink to={paths.about} key={2}>
+      <Trans>About</Trans>
+    </LocaleNavLink>,
+    <LocaleNavLink to={paths.howToUse} key={3}>
+      <Trans>How to use</Trans>
+    </LocaleNavLink>,
+    <a href="https://www.justfix.nyc/donate" key={4}>
+      <Trans>Donate</Trans>
+    </a>,
+  ];
```

Comment:
Rather than returning an array, which requires you to add a `key` prop to each child, you could return a [React fragment](https://reactjs.org/docs/fragments.html) with e.g.:

```jsx
return <>
  <p>child one</p>
  <p>child two</p>
  <p>etc...</p>
</>;
```

Reply:
hmmm so the issue here is that I want to be able to iterate over these link elements in a `map()` function later, so I can place them within menu item components. Is it possible to do that with a react fragment?

Reply:
Oh I didn't realize you wanted to iterate over them--yeah if you want to iterate over them, keep this as-is!